RockLib.Compression/CompressionExtensions.cs

```diff
@@ -14,9 +15,11 @@ public static class CompressionExtensions
         /// <param name="compressor">Compressor to extend</param>
         /// <param name="text">The text to compress</param>
         /// <param name="encoding">The optional encoding to use.</param>
-        /// <returns></returns>
+        /// <returns>The compressed byte array.</returns>
+        /// <exception cref="ArgumentNullException"></exception>
         public static byte[] Compress(this ICompressor compressor, string text, Encoding encoding = null)
         {
+            if (compressor == null) throw new ArgumentNullException(nameof(compressor));
```

Comment:
We should start using `is` for `null` comparisons, like this:
```
if (compressor is null)
```
Using `==` is fine, but it calls that operator, and that can be changed in a type definitions. It's really rare to see anyone ever change it to do something nefarious with `null`, but using `is` will always do a direct `null` comparison.

Reply:
👍

Reply:
changed.